components/content-service/pkg/initializer/initializer.go

```diff
@@ -191,10 +191,10 @@ func (bi *fromBackupInitializer) Run(ctx context.Context, mappings []archive.IDM
 
 	hasBackup, err := bi.RemoteStorage.Download(ctx, bi.Location, storage.DefaultBackup, mappings)
 	if !hasBackup {
-		return src, xerrors.Errorf("no backup found")
+		return src, fmt.Errorf("no backup found: %v", err)
 	}
 	if err != nil {
-		return src, xerrors.Errorf("cannot restore backup: %w", err)
```

Comment:
why the `xerrors` change?

Reply:
If we use `xerrors`, we get a warning that we have to use `fmt` instead of it.

Reply:
@utam0k I made a similar change and got this comment https://github.com/gitpod-io/gitpod/pull/10339#discussion_r884669484

https://github.com/gitpod-io/go-style-guide/blob/master/style.md#errors

Reply:
@aledbf Thanks for your information. I haven't seen it. You are right.

Reply:
Hm. I was under the impression that we will be slowly switching to fmt instead of xerrors.
@easyCZ I think it was in one of your PRs that this was discussed, unless I am mistaken.

Reply:
> Hm. I was under the impression that we will be slowly switching to fmt instead of errors.

Same here. Maybe the guide is not clear about that?

Reply:
@csweichel ^

Reply:
[A related Slack thread](https://gitpod.slack.com/archives/C01KGM9AW4W/p1651840589492279)

Reply:
> Same here. Maybe the guide is not clear about that?

The guide is clear IMHO: it promotes `xerrors` (on purpose).


Thanks @kylos101 for linking the Slack thread. We didn't really land on a change, because
- `fmt.Errorf` neither carries, nor perpetuates a stack trace
- `github.com/pkg/errors` would produce a stack trace, but is deprecated and yet another third party library

xerrors is not ideal, but 
- it works
- it is reasonably consistent throughout our codebase. Until we decide to do something else we should keep it this way to ease migration "search + replace" style.
- it's not deprecated
- it comes from the Go team themselves.

All that said, `xerrors` also isn't exactly future proof, and we could look for alternatives **in an RFC**.

Reply:
updated PR.